Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.9 as builder
+RUN go get -d github.com/newrelic/nri-mongodb/... && \
+    cd /go/src/github.com/newrelic/nri-mongodb && \
+    make && \
+    strip ./bin/nr-mongodb
+
+FROM newrelic/infrastructure:latest
+COPY . .
```

Comment:
why do we need to copy the whole context if we are downloading the code with go in the builder stage?

Reply:
I forgot that from testing. Not needed, thanks for pointing it out